### DISCUSSION TITLE
How do i connect to server?

### DISCUSSION THREAD

**Discussion by snip3shock** [O365/python-o365#667] [Q&A]
Hi I am looking at the documentation type and have managed to build with the installer sh successfully.

May I know how I can test the sending of message as the documentation is not very detailed in steps. 
I am not sure where I can add the code snippet or run the program as shown in the link.
Would appreciate any help, thanks in advance!

https://pypi.org/project/O365/#authentication

**Comment by alejcas** [comment]
https://github.com/O365/python-o365#table-of-contents
Follow Install, Usage and Authentication and you're good to go
